app/models/user.rb

```diff
@@ -1,5 +1,9 @@
 class User < ApplicationRecord
-  has_many :posts
+  has_many :posts, foreign_key: 'author_id'
   has_many :comments
   has_many :likes
+
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
 end
```

Comment:
- Hey, please, kindly check your migration files, there's one which is throwing this error below: kindly fix this issue in your database:


![image](https://github.com/Lul-Abdifan/Blog-App/assets/59397159/b9a5c2d8-7c75-497f-b904-8dfc07bfc92c)

Reply:
@zieeco I am not getting your feedback. Could you please specify the problem you got when reviewing?

Reply:
@Lul-Abdifan, I can't run the migration on your database, that's the error it throws. check  [this migration file](https://github.com/Lul-Abdifan/Blog-App/blob/88fc7cc57ae6a68bc8295871ec57f3a56c12318f/db/migrate/20230720133220_rename_old_table_name_to_new_table_name.rb#L1-L5) to fix it